src/main/resources/db/migration/common/V1_224__V50_V51_V52.sql

```diff
@@ -0,0 +1,50 @@
+ALTER TABLE courtroom DROP COLUMN last_modified_ts;
```

Comment:
What about region, object_directory_status, external_location_type tables?
They also look to need columns dropping - if you can double check please

Reply:
Please double check your local postgres / commit 743275bc618ec8191ed627b5d2c0b477cfa2cd69
e.g. these 4 columns removed, marked -
``` CREATE TABLE region
 (reg_id                      INTEGER                       NOT NULL
 ,region_name                 CHARACTER VARYING             NOT NULL
-,created_ts                  TIMESTAMP WITH TIME ZONE      NOT NULL
-,created_by                  INTEGER                       NOT NULL
-,last_modified_ts            TIMESTAMP WITH TIME ZONE      NOT NULL
-,last_modified_by            INTEGER                       NOT NULL
 ) TABLESPACE darts_tables;
 ```

Reply:
ACCEPT. I was getting those confused with the same named columns.